Header header
float32 leftAngle
float32 leftVel
float32 rawLeftDelta
float32 rightAngle
float32 rightVel
float32 rawRightDelta
